types/opentok/index.d.ts

```diff
@@ -69,6 +70,13 @@ declare module 'opentok' {
       hasVideo: boolean;
     }
 
+    export interface PatchStream {
+      addStream?: string;
+      removeStream?: string;
+      hasAudio: boolean;
+      hasVideo: boolean;
```

Comment:
I think these properties are meant to be optional (<https://tokbox.com/developer/rest/#patch-data>)

```suggestion
      hasAudio?: boolean | undefined;
      hasVideo?: boolean | undefined;
```